rs_server/CADIP/api/cadu_download.py

```diff
@@ -223,10 +226,10 @@ def download(
         db_product = CaduDownloadStatus.get(db, name=name)
     except HTTPException as exception:
```

Comment:
Here CaduDownloadStatus.get would raise a HTTPException 404 if the product is missing from the database (don't know if it could happen here). If we lost connection, it would rather raise a sqlalchemy.exc.OperationalError.

(I don't know if we should document all the exceptions that could be raised by all the functions. It would be difficult to maintain this docstring).

But anyway below you're raising a HTTP_503_SERVICE_UNAVAILABLE in case of HTTPException 404, I don't know if it's relevant, and you're not catching sqlalchemy.exc.OperationalError. But do we want to make the effort of catching all exceptions and returning {"started": "false"} ? Can't we let the exception propagate, let the caller catch it and set {"started": "false"} in case of any exception ?

Reply:
Agree, i'll update to catch a more general exception.

About HTTP_503_SERVICE_UNAVAILABLE, it should still be raised, because download service is unavailable if any subsystem fails.